GSA/digitalgov.gov
# Requirements Checklist resource: update checklist of requirements

## Summary

Updates the checklist of requirements to a more current and readable format. Removes many "lists of links." 

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/bc-checklist-update/resources/checklist-of-requirements-for-federal-digital-services)



### Solution

### How To Test

1. Make sure the formatting is true to the draft document
2. Make sure there are no typos
3. Check all links

---

### Dev Checklist
